src/common/actions/snaps.js

```diff
@@ -12,14 +12,16 @@ export const REMOVE_SNAP = 'REMOVE_SNAP';
 export const REMOVE_SNAP_SUCCESS = 'REMOVE_SNAP_SUCCESS';
 export const REMOVE_SNAP_ERROR = 'REMOVE_SNAP_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps(owner, showFetching = true) {
```

Comment:
Not sure if I agree with such approach. This makes the state in store little bit inconsistent, because if we call it with `showFetching = false` data is fetching (we have request in progress), but the store doesn't reflect this change.

If we don't want to show spinner during polling (and I agree with that) we should make UI make this decision based on state in the store - not make the store pretend there is no request in progress.

In build pages (where we poll for builds statuses) it's done as `showLoading = isFetching && !success` - but this assumes that action that inits fetching doesn't clear the success status in store. This is not the case in `snaps` reducer as far as I can see. And I guess it's debatable what should be our convention in such case (should we clear success when we set isFetching)?

Other option would be do do `showLoading = isFetching && !snaps`. So we will only show loading if we don't have snaps data already.

So, to summarize my point here: I think the state should always have isFetching set when we wait for response (for example so we can detect it and not fetch again until completed), and it's UI component that should decide about showing loading based on what is in the store. I don't think we should create 'silent' action that fetch requests, but hide some info from the store.